store/sqlstore/team_store.go

```diff
@@ -643,6 +643,7 @@ func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restricti
 	query := s.getTeamMembersWithSchemeSelectQuery().
 		Where(sq.Eq{"TeamMembers.TeamId": teamId}).
 		Where(sq.Eq{"TeamMembers.DeleteAt": 0}).
+		OrderBy("UserId").
```

Comment:
This is outside of my scope of domain knowledge, but just want to make sure the teams relying on this API call weren't implicitly assuming anything about the previous order. It would have closely approximated `CreateAt` previously, and will now be effectively random. 

@jespino, I seem to recall your being involved with permissions on platform/server -- can you confirm this is safe for clients of this API?

Reply:
There is a test in the app layer that is clearly defining the order as ordered by UserId (see the referenced code in the description of the PR). And that have no relationship with the creation date because we are using a base32 encoded version of the an UUID4 random value, so it shouldn't have any relationship with the date.

Reply:
Ah, perfect -- I couldn't quite parse the original comment.

My expectation re: the creation date is that, in some cases, the natural sort order returned matches the order in which the rows were written to disk. Strictly speaking, this isn't guaranteed, but would closely approximate `CreateAt`. Perhaps this was the order Postgres was defaulting to, and maybe MySQL's defaults converge differently.

Thanks for clarifying!

Reply:
The default mysql looks like was by primary key, but anyway was accidentally ordered, so wasn't ensured, and with pagination, consistent and reliable order is a must.